tests/test_admin.py

```diff
@@ -134,24 +132,29 @@ def test_poll_version_is_not_added_for_change_true(self):
         pc2 = factories.PollContentFactory()
         request = RequestFactory().get('/admin/polls/pollcontent/')
         model_admin.save_model(request, pc2, form=None, change=True)
-        check_obj_exist = PollVersion.objects.filter(content=pc2).exists()
+        check_obj_exist = Version.objects.filter(
+            content_type=ContentType.objects.get_for_model(pc2),
+            object_id=pc2.pk).exists()
         self.assertFalse(check_obj_exist)
 
     def test_blogpost_version_is_added_for_change_false(self):
         model_admin = self._get_admin_class_obj(BlogContent)
-        bc1 = factories.BlogContentFactory()
+        pc2 = factories.BlogContentFactory()
```

Comment:
Why is this getting changed here? bc would stand for blog content, but what would pc stand for in this case?